README.md

```diff
@@ -24,24 +32,135 @@ The optimal choice and combination of bridging species may vary from one genomic
+  -q, --quiet           Do not produce any log output (default: False)
+  -v, --verbose         Produce additional debugging output (default: False)
+  -c, --simple_coords   Make coord numbers in debug output as small as possible (default: False)
+  -a, --include_anchors Include anchors in results table (default: False)
+```
+
+
+## Required Input
+
+In addition to a `.bed` file containing genomic regions of interest from a reference species (e.g. mm39), the other required input for IPP is a `.pwaln` file, which is a binarized collection of pairwise alignments between the reference, target, and all bridging species.
+We provide a set of precomputed `.pwaln` files for selected comparisons across vertebrate species. The set of bridging species used for these files are the same as those described in our [preprint](https://www.biorxiv.org/content/10.1101/2024.05.13.590087v1). The provided collection includes files for comparisons where mouse (mm39), human(hg38) and chicken (galGal6) serve as the reference genomes. These large files are stored separately from github and can be downloaded [HERE](https://owww.molgen.mpg.de/~IPP/)
+
+### Generate custom alignments
+We provide a Snakemake pipeline to compute your own alignment collections for your choice of species. 
+   For that, run `compute_alignments/compute_pairwise_alignments`. The script will guide you through the whole alignment process from fasta to chain files. 
+   Make sure all dependencies are installed, including `LAST` and utilities to handling chain files from [UCSC](https://hgdownload.soe.ucsc.edu/admin/exe/): `axtChain`, `chainMergeSort`, and `chainPreNet`. 
+
+```
+Usage: compute_pairwise_alignments -s SPECIES -t TARGETS [optional: -d DATA_DIR -f FORCE -@ NTHREADS -n DRY_RUN]
+
+required (one or the other):
+  -s SPECIES      Comma-separated list or file (first column) with species (genome builds) for which to compute pairwise alignments
+  -t TARGETS      Comma-separated list or file with target files (e.g. a list of chain files) to be produced
+
+optional:
+  -c CREATE_PWALN_COLLECTION		Flag for creating the pwaln collection file. Used as input for IPP
+  -r REFERENCE		Reference species. Necessary for naming the collection file
+  -q QUERY			Query species. Necessary for naming the collection file. If no query is provided, will generate for all non-reference species. 
+  -d DATA_DIR   		Output directory. The script will create a predefined folder structure in that directory. (e.g. separate folders for alignment files, fasta files, etc.)
+  -f FORCE			Force execution and overwriting existing files. Possible values: none (default) | space-separated rule names | all
+  -@ NTHREADS     	Number of parallel threads
+  -n DRY_RUN      	Flag for a dry run, i.e. only print the commands instead of executing
+  -! SNAKEMAKE_ARGS		Optional arguments for Snakemake in quotes, i.e. -! "--debug-dag --unlock -p"
+```
```

Comment:
```suggestion
Move to the compute_alignments folder and run `compute_pairwise_alignments -h` for a detailed description of the arguments.
```
Also here, I suggest to remove the duplicated shell output.

Reply:
same as above